charts/sonarqube/values.yaml

```diff
@@ -519,10 +519,10 @@ extraConfig:
   secrets: []
   configmaps: []
 
-# account:
+account:
 # The values can be set to define the current and the (new) custom admin passwords at the startup (the username will remain "admin")
-#   adminPassword: admin
-#   currentAdminPassword: admin
+  adminPassword: admin
```

Comment:
Do we want to enable the password change by default ? it will fail for following upgrades as the current password would have been changed by a user in the ui ?